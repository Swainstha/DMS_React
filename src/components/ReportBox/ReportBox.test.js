import testSnapFunction from '../../util/testSnapFunction';
import ReportBox from './index'
import React from 'react';
const dummy_data= {
 styleName:'', 
 icon:"icon1.png", 
 price:32, 
 detail:"no more details", 
}

testSnapFunction("<ReportBox />", "Snapshot test for ReportBox ",<ReportBox {...dummy_data}><div>dsjfM</div> </ReportBox> )