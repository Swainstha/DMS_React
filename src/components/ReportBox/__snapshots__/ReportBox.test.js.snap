// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReportBox /> Snapshot test for ReportBox  1`] = `ShallowWrapper {}`;
